### PR TITLE
NUI-5999: wizard-step.component.ts overwrite 'steps' class property f…

### PR DESCRIPTION
**Pull request by DmytroPutrin**
…or WizardDirective

### CODE REVIEW

File: packages/bits/src/lib/wizard-v2/wizard-step.component.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ export class WizardDirective extends CdkStepper implements AfterContentInit {
 
     /** Steps that the stepper holds. */
     @ContentChildren(WizardStepV2Component, {descendants: true}) _steps: QueryList<WizardStepV2Component>;
+    @ContentChildren(WizardStepV2Component, {descendants: true}) steps: QueryList<WizardStepV2Component>;
```
Comment (Blakesalot) [inline packages/bits/src/lib/wizard-v2/wizard-step.component.ts:101]: Do we need two of these properties? `_steps` and `steps`

Comment (DmytroManco) [reply packages/bits/src/lib/wizard-v2/wizard-step.component.ts:101]: yes. We wrap CDK Stepper component, that use both property and refs to CDK Step component. If we want to use our custom WizardStepV2Component, we should overwrite QueryList.
https://github.com/angular/components/blob/master/src/cdk/stepper/stepper.ts#L268 and line 271

Comment (Blakesalot) [reply packages/bits/src/lib/wizard-v2/wizard-step.component.ts:101]: Oh, wait. It looks like this `steps` property is overriding the parent CdkStepper `steps` property. I'm not sure that this is a good thing.

Comment (Blakesalot) [reply packages/bits/src/lib/wizard-v2/wizard-step.component.ts:101]: Okay, do you mind adding an inline comment about that to help prevent confusion in the future?

Comment (DmytroPutrin) [reply packages/bits/src/lib/wizard-v2/wizard-step.component.ts:102]: Comment added.

**Review by ivankozlovskyi — APPROVED** [review]

**Review by dmytrognatyshyn — APPROVED** [review]

**Review by Vita-Karas — APPROVED** [review]

**Pull request merged**